portfolio/src/main/webapp/contact.html

```diff
@@ -0,0 +1,41 @@
+<!DOCTYPE html>
+<meta charset="UTF-8">
+<title>Roland's - Portfolio</title>
+<link rel="stylesheet" href="style.css">
+<script src="script.js"></script>
+
+<nav id="navbar">
```

Comment:
Common practice is to use classes instead of IDs where possible (IDs can be important for JS and anchor links, but all CSS can be done using classes).  The distinction is more important on larger sites, where there are many layers of templates and it's difficult to know for sure that your ID is unique